x-pack/test/cloud_security_posture_functional/pages/findings.ts

```diff
@@ -189,19 +188,6 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
       });
     });
 
-    describe('DistributionBar', () => {
```

Comment:
So we are now getting more coverage using integration (MSW) than FTRs!

Reply:
Yes!!